modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/parse/MBExpressionParseTest.java

```diff
@@ -157,7 +158,52 @@ public void testParseUpcaseConcat() {
     }
 
     // ---- COLOR EXPRESSIONS ---------------------------------------------------------
+    @Test
+    public void testParseRgb() {
+
+        JSONObject layer = testLayersById.get("rgbExpression");
+        Optional<JSONObject> o = traverse(layer, JSONObject.class, "paint");
+        JSONObject j = o.get();
+        assertEquals(JSONArray.class, j.get("text-color").getClass());
+        JSONArray arr = (JSONArray) j.get("text-color");
+        assertEquals(MBColor.class, MBExpression.create(arr).getClass());
+        Expression rgb = MBExpression.transformExpression(arr);
+        Object c = rgb.evaluate(rgb);
+        assertEquals(new Color(0, 111, 222), c);
+    }
```

Comment:
Test looks good, although an additional SLD conversion test to verify that it gets converted to #RRGGBB format might be in order (it _should_ just work, but its still good to test)

Reply:
SLD conversion test added, confirmed rgb function holds up to the conversion, unsure best way to verify #RRGGBB format.

Reply:
As long as the SLD converts correctly, it should get encoded. If you want to be extra careful, you can write the SLD out as a string (via `String xml = new SLDTransformer().transform(sld)`) and do a string contains check, but what you've added should be sufficient.

Reply:
The SLDTransformer doesn't generate the #RRGGBB transformation from the function, but the string xml writes out the functions as expected.

Reply:
Right, I guess that actually makes sense. Seems like everything is functioning as expected at least.